content/blog/scales-1-2-0/index.Rmd

```diff
@@ -0,0 +1,174 @@
+---
+output: hugodown::hugo_document
+
+slug: scales-1-2-0
+title: scales 1.2.0
+date: 2022-04-12
+author: Hadley Wickham
+description: >
+    scales 1.2.0 brings a number of small but useful improvements 
+    to numeric labels.
+
+photo:
+  url: https://unsplash.com/photos/98MbUldcDJY
+  author: Piret Ilver
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [package] 
+tags: [ggplot2, scales]
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(fig.asp = 1/3)
+ggplot2::theme_set(ggplot2::theme_gray(18))
+set.seed(1014)
+```
+
+
+```{=html}
+<!--
+TODO:
+* [x] Look over / edit the post's title in the yaml
+* [x] Edit (or delete) the description; note this appears in the Twitter card
+* [x] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [ ] Find photo & update yaml metadata
+* [x] Create `thumbnail-sq.jpg`; height and width should be equal
+* [x] Create `thumbnail-wd.jpg`; width should be >5x height
+* [x] `hugodown::use_tidy_thumbnails()`
+* [x] Add intro sentence, e.g. the standard tagline for the package
+* [x] `usethis::use_tidy_thanks()`
+-->
+```
+
+We're very pleased to announce the release of [scales](https://scales.r-lib.org) 1.2.0.
+The scales package provides much of the infrastructure that underlies ggplot2's scales, and using it allow you to customize the transformations, breaks, and labels used by ggplot2.
+You can install it from CRAN with:
+
+```{r, eval = FALSE}
+install.packages("scales")
+```
+
+This blog post will show off a few new features for labeling numbers, log scales, and currencies.
+You can see a full list of changes in the [release notes](https://github.com/r-lib/scales/blob/main/NEWS.md).
+
+```{r setup}
+library(ggplot2)
+library(scales)
+```
+
+## Numbers
+
+`label_number()` is the workhorse that powers ggplot2's formatting of numbers, including `label_dollar()` and `label_comma()`.
+This release added a number of useful new features.
+
+The most important is a new `scale_cut` argument that makes it possible to independently scales different parts of the range.
+This is useful for scales that span multiple orders of magnitude. Take the following two examples which don't get great labels by default:
+
+```{r}
```

Comment:
Add alt-text for figures

Reply:
Exactly, just describe what's different about the portion that you're highlighting